package.json

```diff
@@ -4,15 +4,16 @@
   "description": "Experimental native Web Components compiler.",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ProjectEvergreen/wcc.git"
+    "url": "git+https://github.com/ProjectEvergreen/wcc.git"
   },
   "type": "module",
   "main": "src/wcc.js",
   "types": "./src/index.d.ts",
   "exports": {
     ".": {
+      "types": "./src/index.d.ts",
       "import": "./src/wcc.js",
-      "types": "./src/index.d.ts"
+      "require": "./src/wcc.js"
```

Comment:
I don't think we need this `require`, I don't need it to make publint pass as long as the `types` come first